lib/modules/datasource/docker/index.ts

```diff
@@ -1,4 +1,4 @@
-import URL from 'url';
+import URL from 'node:url';
```

Comment:
we should get rid of this deprecated module

Reply:
Tracked in #7797